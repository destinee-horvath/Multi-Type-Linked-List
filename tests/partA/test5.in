NEW 5
1.23e-0
1.23e-1
1.23e-2
1.23e-3
1.23e-4
VIEW 0
TYPE 0
NEW 4
many
spaces            
end
VIEW 1
VIEW ALL