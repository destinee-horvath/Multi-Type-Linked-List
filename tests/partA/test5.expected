List 0: 1.23 -> 0.12 -> 0.01 -> 0.00 -> 0.00
1.23 -> 0.12 -> 0.01 -> 0.00 -> 0.00
float -> float -> float -> float -> float
List 1: many -> spaces             -> end -> VIEW 1
Number of lists: 2
List 0
List 1
